app/styles/app.css

```diff
@@ -1178,6 +1178,8 @@ input#search:-ms-input-placeholder {
 body * { visibility: hidden; }
 #printcontent * { visibility: visible; }
 #printcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
+    #printreceiptcontent * { visibility: visible; }
+    #printreceiptcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
```

Comment:
since the classes introduced seem to be identical to existing print content classes, is there really a need to add the new classes?